brave/brave-ios
# Fix #2218: Move monthly contributions into its own section and list

## Summary of Changes

This pull request fixes issue #2218 

## Submitter Checklist:


## Test Plan:

- Enable rewards, verify monthly contributions section now exists in settings screen
- Make a monthly tip, verify said publisher appears in the monthly contributions screen and do not appear in the tips screen. 
- Verify no date appears on recurring tips rows
- Verify date appears for one time tips (the date you tipped)
- Verify total tips on monthly contributions screen equals the amount you will tip monthly to all publishers
- Verify total tips on tips screen equals the amount of bat you've tipped already to publishers this month

## Screenshots:

| Monthly Contributions | Tips |
| --- | --- |
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-01-22 at 17 53 19](https://user-images.githubusercontent.com/529104/72942295-1011ec00-3d41-11ea-9e2d-8a0b7e5ee95f.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-01-22 at 17 53 24](https://user-images.githubusercontent.com/529104/72942369-3a63a980-3d41-11ea-97e5-576ac42d0c2a.png) |

![Simulator Screen Shot - iPhone 11 Pro - 2020-01-22 at 18 04 48](https://user-images.githubusercontent.com/529104/72942537-b52cc480-3d41-11ea-8fc2-b193cfa78108.png)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
